repos/spack_repo/builtin/packages/malt/package.py

```diff
@@ -22,34 +22,71 @@ class Malt(CMakePackage):
     license("CECILL-C")
 
     # Versions
+    version("1.4.0", sha256="fb64e99eec9b9d3cb46b5f9cbd1e47b31354356ebc0502c27af101dfcff68b9f")
+    version("1.3.1", sha256="9f3b22ace13e0a3bc773fed4044a5c19439aeb9111077582704382b3d1675194")
+    version("1.3.0", sha256="7b19db65dd1d95a84c06676c31059312ff11a6e65cbba32b160370d0e1dad33b")
     version("1.2.5", sha256="9660e42f92230e6acf5c19df5195f59a4c2d7d919eeab4410fe943507eee2c67")
-    version("1.2.4", sha256="47068fe981b4cbbfe30eeff37767d9057992f8515106d7809ce090d3390a712f")
-    version("1.2.3", sha256="edba5d9e6a11308f82b9c8b61871e47a8ae18493bf8bff7b6ff4f4a4369428de")
-    version("1.2.2", sha256="543cace664203fd9eb6b7d4945c573a3e507a43da105b5dc7ac03c78e9bb1a10")
+    version(
+        "1.2.4",
+        sha256="47068fe981b4cbbfe30eeff37767d9057992f8515106d7809ce090d3390a712f",
+        deprecated=True,
+    )
+    version(
+        "1.2.3",
+        sha256="edba5d9e6a11308f82b9c8b61871e47a8ae18493bf8bff7b6ff4f4a4369428de",
+        deprecated=True,
+    )
+    version(
+        "1.2.2",
+        sha256="543cace664203fd9eb6b7d4945c573a3e507a43da105b5dc7ac03c78e9bb1a10",
+        deprecated=True,
+    )
```

Comment:
Here we can use:
```python
with defatult_args(deprecated=True):
```
you can find examples in other recipes.

Should we also deprecate v1.2.1 ? Or is there some reason to keep it?